docs/CRAWLING.md

```diff
@@ -0,0 +1,239 @@
+  "pushed_at": "2020-10-08T12:18:22Z",
+  "stargazers_count": 136,
+  "watchers_count": 136,
+  "language": "JavaScript",
+  "forks_count": 331,
+  "open_issues_count": 98,
+  "license": null,
+  "default_branch": "master",
+  "owner": {
+    "login": "Sol",
+    "avatar_url": "./images/demo/Sol.png"
+  }
+},
+```
+
+To retrieve this information from your GitHub instance, query the [GitHub Search API](https://developer.github.com/v3/search/) for all projects with the topic `inner-source`.
+
+   ```
+   ?q=topic:inner-source
+   ```
+
+You can write the API response to `repos.json` as is, as the field names above are the same as returned by the GitHub API. Additional fields returned by the GitHub API will be ignored by the portal.
+
+ > *Note:* This assumes that all projects that you want to display have the topic `inner-source`. You can of course use any other topic as well. You can optionally limit the results further  by adding `is:public` or `is:private` to the query, depending on how InnerSource repositories are characterized in your environment.
+
+### Step 2: (Optional) Extended Project Information from GitHub
+
+For a richer portal experience, you can add extended project information that is available via further GitHub API calls.
+
+For each project retrieved in step (1), add a key `_InnerSourceMetadata` with an object containing the following metadata about the project:
+
+* `topics` (Optional): Query GitHub [topics](https://docs.github.com/en/rest/reference/repos#get-all-repository-topics) and add the array of topics with the key `topics` for each repo to allow searching projects by topic and displaying them on the detail popup.
+* `participation`: Query GitHub for the [weekly commit count](https://docs.github.com/en/free-pro-team@latest/rest/reference/repos#get-the-weekly-commit-count) (subset "all") and add it. with the key `participation`
+* `guidelines` (Optional): Check if there are contribution guidelines and add the file name with the key `guidelines` (e.g. `CONTRIBUTING.md`). If specified, the *Contribute* button for this project will link to this file instead of the repository root.
+
+### Step 3: (Optional) Custom Data
+
+You can customize the portal further with the following properties, that you can add to the `_InnerSourceMetadata` object explained in step (2):
+
+* `score` (Optional): Calculate the [Repository Activity Score](https://patterns.innersourcecommons.org/p/repository-activity-score) to define a meaningful order for the projects. Add it with the key `score`.
+* Check if there is a file `innersource.json` in the repository and add all keys from that file directly below `_InnerSourceMetadata`. For further details review the [syntax definition](LISTING.md#syntax-definition-of-innersourcejson) of `innersource.json`.
+
+## Reference Implementations
+
+You will have to adapt all of these crawler implementations to your exact setup. However they may give you a good starting points.
+
+* with curl/jq: TBD SAMPLE HERE
```

Comment:
would be great to have the 3 minute example here, i played around with it yesterday but did not get it working immediately